includes/rest-api/class-sensei-rest-api-setup-wizard-controller.php

```diff
@@ -466,6 +458,17 @@ public function submit_tracking( $data ) {
 		Sensei()->usage_tracking->set_tracking_enabled( (bool) $data['tracking']['usage_tracking'] );
 		Sensei()->usage_tracking->send_usage_data();
 
+		$setup_purpose_data = $this->setup_wizard->get_wizard_user_data( 'purpose' );
+		if ( $setup_purpose_data ) {
+			sensei_log_event(
+				'setup_wizard_purpose_continue',
```

Comment:
I'm a bit hesitant because we have some jobs running (see p4qSXL-5zo-p2_missed_partitions-hdv-check/#comment-17945) that rely on this job name. It also might be used for historical comparisons.

Reply:
Ah, so I think that's fine! :)
It has a little of true because it's sending the purpose data.. haha